### DISCUSSION TITLE
Polaris-tokens bundle size reduce - built files include unnecessary data

### DISCUSSION THREAD

**Discussion by juzser** [Shopify/polaris-react#11807] [General]
Hi there, 
As I see the bundle of polaris-tokens is large, after investigated, I see there are a lot of data are unnecessary and actually unused anywhere. 

The whole ideas of the polaris-tokens should be the variables & tokenization only, it should not contains any content or rendering copy data. If the dev team can wipe out those data, we can save a bit of package size. 

For example in the `polaris-tokens/src/themes/base/`,  many files contain the description just to be used in the docs (I guess). 
`color.ts` file [here](https://github.com/Shopify/polaris/blob/main/polaris-tokens/src/themes/base/color.ts#L256) is an example. 

And the root index file that includes the whole `base/` files, will be built as module and will contain those copy inside the bundle.

I think the dev team should bring the content to other directory or data files, and exclude them from `polaris-tokens` built bundle.

**Comment by daviareias** [comment]
I noticed the same things and all of these tokens are also in the css files as well, so I don't know whats the point
